cmd/api/src/database/migration/migrations/v8.3.0.sql

```diff
@@ -120,3 +120,47 @@ WHERE agts.id = duplicate_selectors.id AND duplicate_selectors.rowNumber > 1;
 ALTER TABLE IF EXISTS asset_group_tag_selectors DROP CONSTRAINT IF EXISTS asset_group_tag_selectors_unique_name_asset_group_tag;
 ALTER TABLE IF EXISTS asset_group_tag_selectors ADD CONSTRAINT asset_group_tag_selectors_unique_name_asset_group_tag UNIQUE ("name",asset_group_tag_id,is_default);
 
+-- Update RO-DC default selector within Tier Zero to use the correct attribute name
+WITH src_data AS (
```

Comment:
This selector already exists everywhere, so this should be an update statement just updating the cypher where name = 'Read-Only DCs' and `is_default` = true